### PR TITLE
Add dropdown input value

### PR DESCRIPTION
**Pull request by janrembold**
This branch 
- updates to Storybook 5.0.11
- removes import/export index.js 
- refactores all imports from index.js to standard compliant imports
- makes the Dropdown component controllable

- [x] Test added / Snapshots updated
- [x] Documentation added / updated

### CODE REVIEW

**Comment by janrembold** [comment]
@mosch please firmly look at this change: https://github.com/allthings/elements/pull/805/files#diff-eb89267f16714776a393d0f85ea63cb5R101 -> Tests were failing here and changing this resolves the tests but it feels strange to change something important.

File: src/organisms/Dropdown.jsx [hunk 1]
```diff
@@ -105,16 +115,21 @@ export default class Dropdown extends React.PureComponent {
...
       items,
       initialSelectedItem,
+      selectedItem,
       placeholder,
       onSelect,
       name,
     } = this.props
     const { showScrollArrow } = this.state
+
+    console.log('render selectedItem', selectedItem)
```
Comment (mosch) [inline src/organisms/Dropdown.jsx]: Let's clean that up first 😇

File: src/organisms/Dropdown.jsx [hunk 2]
```diff
@@ -154,7 +169,7 @@ export default class Dropdown extends React.PureComponent {
                       overflow: 'hidden',
                       whiteSpace: 'nowrap',
                       cursor: 'pointer',
-                      paddingRight: '30px',
+                      paddingRight: '35px',
```
Comment (mosch) [inline src/organisms/Dropdown.jsx]: Why was that changed?

File: stories/DropdownStory.jsx [hunk 3]
```diff
@@ -43,6 +54,7 @@ class DropdownStory extends React.Component {
   }
 
   render() {
+    console.log('render', this.state)
```
Comment (mosch) [inline stories/DropdownStory.jsx]: Let's clean  that up

File: src/organisms/Dropdown.jsx [hunk 4]
```diff
@@ -52,11 +57,16 @@ export default class Dropdown extends React.PureComponent {
         value: PropTypes.any.isRequired,
       })
     ).isRequired,
-    /** Initially selected */
+    /** Initially selected item - this value is uncontrolled */
```
Comment (mosch) [inline src/organisms/Dropdown.jsx]: React used the prop _defaultValue_ for this behaviour. I think it would be a good idea to make that consistent.

File: src/organisms/Dropdown.jsx (hunk 1 above)
Comment (janrembold) [reply src/organisms/Dropdown.jsx]: Damn, too much debugging during this "little" task. Is removed now

File: stories/DropdownStory.jsx (hunk 3 above)
Comment (janrembold) [reply stories/DropdownStory.jsx]: Done

File: src/molecules/Collapsible.jsx [hunk 5]
```diff
@@ -1,8 +1,10 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import { css } from 'glamor'
-import { View, Icon, Text } from '../'
 import { ColorPalette, alpha } from '@allthings/colors'
+import Icon from '../atoms/Icon'
```
Comment (dan-lee) [inline src/molecules/Collapsible.jsx]: All the `import` changes shouldn't be needed anymore. At least it worked in my branch without any problems.

File: src/organisms/Dropdown.jsx (hunk 2 above)
Comment (janrembold) [reply src/organisms/Dropdown.jsx]: The text is too long and the dots of the ellipsis effect collided with the drowdown arrow

File: src/organisms/Dropdown.jsx (hunk 4 above)
Comment (janrembold) [reply src/organisms/Dropdown.jsx]: Don't understand. Do you mean renaming the prop `initialSelectedItem` to `defaultValue`?

File: src/molecules/Collapsible.jsx (hunk 5 above)
Comment (janrembold) [reply src/molecules/Collapsible.jsx]: Very funny 😒Changing this took forever.
But what is the advantage of a single-import-export-file anyway?

Comment (dan-lee) [reply src/molecules/Collapsible.jsx]: It's a good indicator that the component has been exported correctly for the consumers. Happened a few times that it wasn't. Afaik it was single file imports and has only been changed lately.

File: src/organisms/Dropdown.jsx (hunk 2 above)
Comment (janrembold) [reply src/organisms/Dropdown.jsx]: Don't know why it worked before, but now the result is still the same. I changed the total width of the input to resolve that issue:

Bildschirmfoto 2019-05-13 um 13 11 58

File: src/molecules/Collapsible.jsx (hunk 5 above)
Comment (janrembold) [reply src/molecules/Collapsible.jsx]: For me it still doesn't work inside Dropdown element for the Icon import

Bildschirmfoto 2019-05-13 um 13 15 35

Comment (janrembold) [reply src/molecules/Collapsible.jsx]: The last error might be resolved with a direct import of Icon

Comment (dan-lee) [reply src/molecules/Collapsible.jsx]: I guess that's because `src/index.js` has been deleted in this PR.

Comment (dan-lee) [reply src/molecules/Collapsible.jsx]: Which of course is not acceptable as it would break everything 😎

Comment (janrembold) [reply src/molecules/Collapsible.jsx]: No, `src/index.js` is there. After I revert everything it still shows this error from the screenshopt above. With Icon as direct import Drowdown works fine

File: src/organisms/Dropdown.jsx (hunk 4 above)
Comment (janrembold) [reply src/organisms/Dropdown.jsx]: This would be a major breaking change and should be done in another branch

Comment (mosch) [reply src/organisms/Dropdown.jsx]: You can keep it backwards compatible in this version and remove it in a later one.
